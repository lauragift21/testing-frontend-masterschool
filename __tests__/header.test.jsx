import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Header from '../src/app/header';

describe('Header', () => {
  it('renders without crashing', () => {
    render(<Header />);
    expect(screen.getByText('Hacker News Clone')).toBeInTheDocument();
  });

  it('creates a snapshot', () => {
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  });
}
);