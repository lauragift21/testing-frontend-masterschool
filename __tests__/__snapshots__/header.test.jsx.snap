// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header creates a snapshot 1`] = `
<DocumentFragment>
  <header>
    <nav
      class="text-lg bg-orange-500 py-2 font-bold"
    >
      <p
        class="navbar-brand"
      >
        Hacker News Clone
      </p>
    </nav>
  </header>
</DocumentFragment>
`;
